raft/server.cc

```diff
@@ -495,13 +510,14 @@ future<add_entry_reply> server_impl::execute_add_entry(server_id from, command c
     if (from != _id && !_fsm->get_configuration().contains(from)) {
         // Do not accept entries from servers removed from the
         // configuration.
-        co_return add_entry_reply{not_a_leader{server_id{}}};
+        co_return add_entry_reply{transient_error{format("Add entry from {} was discarded since "
+                                                         "it is not part of the configuration", from), false, {}}};
```

Comment:
It seems that in this case, the error is neither transient, nor the resulting behavior is desirable: the server will just retry, and even without waiting since `delay = false`.

Reply:
With our implementation it may never be invalidate since we rely on external failure detector as opposite to a heartbeat messages from a leader directly. If a follower will not be aborted after configuration change it may retry forever. We should set delay to true.

Reply:
> We should set delay to true.
done